Compiler warning in StudyGroupsOverview

Frontend compiler warning:

/src/views/studygroups/StudyGroupsOverview.vue
  97:33  error  ["groupID"] is better written in dot notation  dot-notation

--- a/src/views/studygroups/StudyGroupsOverview.vue
+++ b/src/views/studygroups/StudyGroupsOverview.vue
@@ -94,7 +94,7 @@ export default {
   },
   computed: {
     groupID () {
-      return this.$route.params['groupID']
+      return this.$route.params.groupID
     }
   },
   created () {

